src/usb_cam.cpp

```diff
@@ -314,17 +314,19 @@ void uyvy2rgb(char *YUV, char *RGB, int NumPixels)
   }
 }
 
-static void mono102mono8(char *RAW, char *MONO, int NumPixels)
+static void mono102mono8(unsigned char *RAW, char *MONO, int NumPixels)
 {
-  int i, j;
-  for (i = 0, j = 0; i < (NumPixels << 1); i += 2, j += 1)
+  unsigned short *raw16 = (unsigned short*)RAW;
```

Comment:
Instead of using `short` which is only guaranteed to be _at least 16 bits_ use a type like `uint16_t`?